site/content/3.12/aql/high-level-operations/upsert.md

```diff
@@ -63,6 +66,49 @@ UPDATE { logins: OLD.logins + 1 } IN users
 Note that in the `UPDATE` case it is possible to refer to the previous version of the
 document using the `OLD` pseudo-value.
 
+## Filter matching syntax (experimental)
```

Comment:
To the best of my knowledge, this syntax is not experimental. It is a normal part of 3.12.

```suggestion
## Filter matching syntax
```